src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/markup_rope_inputs.hpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/graph_rewrite.hpp"
+
+namespace ov {
+namespace intel_cpu {
+
+class MarkUpRopeInputs: public ov::pass::MatcherPass {
```

Comment:
Could you please add a description to this pass?

Reply:
Sure